packages/evm/CHANGELOG.md

```diff
@@ -161,16 +161,16 @@ This release fully supports the Merge [Kiln](https://kiln.themerge.dev/) testnet
 In the VM the `merge` HF is now activated as being supported and an (experimental) Merge-ready VM can be instantiated with:
 
 ```typescript
-import VM from '@ethereumjs/vm'
-import Common, { Chain, Hardfork } from '@ethereumjs/common'
+import { VM } from '@ethereumjs/vm'
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
 
 const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Merge })
 const vm = new VM({ common })
 vm._common.isActivatedEIP(4399) // true
 ```
 
 - [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399) Support: Supplant DIFFICULTY opcode with PREVRANDAO, PR [#1565](https://
-github.com/ethereumjs/ethereumjs-monorepo/pull/1565)
+  github.com/ethereumjs/ethereumjs-monorepo/pull/1565)
```

Comment:
Ah, no, these kind of format corrections are not correct, this simply destroys the correct rendering and draws the URL apart, see the rendered version of this file.

Please make sure that you haven't updated anywhere else in a similar fashion.